plugins/migration/bin/migrate.js

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview A script for helping to migrate between versions of Blockly.
```

Comment:
I don't know what the purpose of this file is... if we create the subcommand in `rename.js` why can't we add it there? why do we call `parseAndRunMigrations` with no args?

i'm sure there are good reasons :) but could you add them to the comments in this file?

Reply:
> why can't we add it there?

There seems to be disagreement between you and Christopher about whether modules should have side effects or not haha. That's what we did before, but Christopher wanted `rename.js` side-effect-less (which I think is reasonable) so I changed it.

> why do we call parseAndRunMigrations with no args?

It defaults to using `process.argv` if you pass no args. I'll add comments to indicate that!